scripts/main/lychee.js

```diff
@@ -72,16 +152,27 @@ let lychee = {
 	hide_content_during_imageview: false,
```

Comment:
```suggestion
	hide_content_during_imgview: false,
```

This was actually an old typo...